src/TrackableTrait.php

```diff
@@ -38,7 +38,7 @@ trait TrackableTrait
      */
     public function getDesiredName(): string
     {
-        return preg_replace('/.*\\\\/', '', strtolower(get_class($this)));
+        return preg_replace('/.*\\\\/', '', strtolower(static::class));
```

Comment:
is `static::class` somehow better than `get_class($this)`?

Reply:
Yes, it is instantly optimalized by PHP at compile time, changed by one of the default CS rules.